packages/cheqd/src/dids/CheqdDidRegistrar.ts

```diff
@@ -178,6 +196,14 @@ export class CheqdDidRegistrar implements DidRegistrar {
             },
           }
         }
+        // Normalize existing context to an array
```

Comment:
I think it's better to initialize an empty set as there can be some verificationMethods removed. We can compute the entire context after the update

Reply:
`didDocument = options.didDocument` in L186 so the current value of didDocument is from the inputs, so I would keep that same